locales/pt.lua

```diff
@@ -35,4 +37,4 @@ local Translations = {
         ["editor"] = "Later aligator!",
     }
 }
-Lang = Locale:new({phrases = Translations, warnOnMissing = true})
\ No newline at end of file
+Lang = Locale:new({phrases = Translations, warnOnMissing = true})
```

Comment:
The changes in your PR seems fine, although it would be great to perhaps include this change / fix aswell.
```suggestion
if GetConvar('qb_locale', 'en') == "pt" then
    Lang = Lang or Locale:new({phrases = Translations, warnOnMissing = true})
end
```

Reply:
This is not really an required change though (based on the fxmanifest) although it would be to prevent future issues with new translations // preventing future issues when someone adds the proper imports to the fxmanifest (`"locales/*.lua"`)

Reply:
Yeah, I'm behind this, I'd prefer to PR it seperately though?

Reply:
Sure